.tool-versions

```diff
@@ -7,3 +7,4 @@ terraform 1.5.5
 terraform-docs 0.16.0
 terragrunt 0.39.2
 tflint 0.48.0
+python 3.11.5
```

Comment:
We haven't needed this in the .tool-versions file to this point. Are we going to have to add this across the board, or is this something specific to this module?

Reply:
I have removed it. It likely got in there because on my machine there was no global version of python set for asdf